kotlin-grpc/build.gradle.kts

```diff
@@ -0,0 +1,12 @@
+dependencies {
+    implementation(project(":kotlin"))
+    implementation(project(":grpc"))
+    implementation(project(":junit5"))
+
+    // kotlin
+    implementation(libs.kotlin.coroutines.jdk8)
+    implementation(libs.kotlin.reflect)
+
+    testImplementation(libs.reactivestreams.tck)
```

Comment:
It seems not used by the test of this project.

Reply:
When I added the `kotlin-grpc` module, it seems that the configuration of the `kotlin` module was brought in as it was and I couldn't organize it 😓 

In 47db28e, I cleaned up unnecessary depenency and additional module settings.